Userland/Applications/PDFViewer/PDFViewer.h

```diff
@@ -18,7 +18,7 @@ class PDFViewer : public GUI::AbstractScrollableWidget {
     C_OBJECT(PDFViewer)
 
 public:
-    virtual ~PDFViewer() override = default;
+    ~PDFViewer() override = default;
```

Comment:
This looks unrelated to the commit.

Also please don't change this. SerenityOS style is to specify both `virtual` *and* `override`/`final` for readability. :^)